src/change_stream/document.rs

```diff
@@ -0,0 +1,97 @@
+use crate::coll::Namespace;
+use bson::{Bson, Document};
+use serde::{Deserialize, Serialize};
+
+/// An opaque token used for resuming an interrupted `ChangeStream`.
+///
+/// When starting a new change stream, `start_after` and `resume_after` fields on
+/// `ChangeStreamOptions` can be specified with instances of `ResumeToken`.
+///
+/// See the documentation
+/// [here](https://docs.mongodb.com/manual/changeStreams/#change-stream-resume-token) for more
+/// information on resume tokens.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct ResumeToken(pub(crate) Bson);
+
+/// A `ChangeStreamEventDocument` represents a [change event]
+/// (https://docs.mongodb.com/manual/reference/change-events/) in the associated change stream.
```

Comment:
The bracketed text on the above line needs to be on the same line as the URL in order to render properly as a link (running `cargo doc --open` allows you to see how the docs render locally).

Reply:
fixed

Reply:
fixed